src/main/resources/graphql/language.graphqls

```diff
@@ -0,0 +1,42 @@
+enum Language {
+	AFRIKAANS,
+	ALBANIAN,
+	ARABIC,
+	ARMENIAN,
+	BENGALI,
+	BULGARIAN,
+	CHINESE,
+	CZECH,
+	DANISH,
+	DUTCH,
+	ENGLISH,
+	ESTONIAN,
+	FINNISH,
+	FRENCH,
+	GERMAN,
+	GREEK,
+	GUJARATI,
+	ITALIAN,
+	HAWAIIAN,
+	HEBREW,
+	HINDI,
+	JAPANESE,
+	KOREAN,
+	LATIN,
+	POLISH,
+	PORTUGUESE,
+	PUNJABI,
+	RUSSIAN,
+	SANSKRIT,
+	SERBIAN,
+	SINDHI,
+	SPANISH,
+	SWAHILI,
+	SWEDISH,
+	TAMIL,
+	TELEGU,
+	THAI,
+	TURKISH,
+	URDU,
+	WELSH
+}
```

Comment:
As per our style guide, could you please add a newline to the end of this file?

Reply:
Could we add some rule for it in validate itself so we don't have to depend on manually checking this

Reply:
Sorry, what 'validate' are you referring to here?

Reply:
maven validate task

Reply:
Ah, OK. We have Checkstyle set up with validate on the Book Project, so I'll look into using that here. We also have sonar cloud integration that should pick up on that for members